eng/Versions.props

```diff
@@ -36,8 +36,8 @@
     imported. This OK because we want to just have an obvious salt for a local build.
   -->
   <PropertyGroup>
-    <VsixVersionPrefix>17.5.2</VsixVersionPrefix>
-    <AddinMajorVersion>17.5</AddinMajorVersion>
+    <VsixVersionPrefix>17.6.0</VsixVersionPrefix>
```

Comment:
Is this mean to mean "17.6.0" because that is the eventual version this will ship in? Or should it be "17.6.3" to represent preview 3?

Obviously I've guessed at the former, but let me know if you have any more knowledge than me

Reply:
I was assuming it was the latter (17.6.3) but I'm not sure it really matters too much as long as we're consistent.